### PR TITLE
[ISSUE #3825] Do some code optimization.[TopicMetadata]

### LINKED ISSUE #3825: [Enhancement] Do some code optimization.[TopicMetadata]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-storage-plugin/eventmesh-storage-standalone/src/main/java/org/apache/eventmesh/storage/standalone/broker/model/TopicMetadata.java

### Describe the solution you'd like

a. add serialVersionUID
b. add final
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by tooo-bad**
Fixes #3825

### CODE REVIEW

**Comment by Pil0tXia** [comment]
@tooo-bad Please reference other pull requests for a standard format of PR title and `Fixes` keyword.

File: eventmesh-storage-plugin/eventmesh-storage-standalone/src/main/java/org/apache/eventmesh/storage/standalone/broker/model/TopicMetadata.java [hunk 1]
```diff
@@ -25,7 +25,8 @@
  */
 public class TopicMetadata implements Serializable {
 
-    private String topicName;
+    private static final long serialVersionUID = -9011507202771676415L;
+    private final String topicName;
```
Comment (Pil0tXia) [inline eventmesh-storage-plugin/eventmesh-storage-standalone/src/main/java/org/apache/eventmesh/storage/standalone/broker/model/TopicMetadata.java:30]: Better have a newline between these two members to match eventmesh codestyle.

**Review by Pil0tXia — CHANGES_REQUESTED** [review]

**Review by mxsm — COMMENTED** [review]
@tooo-bad  Thanks for your contribution. Please fix CI and link issue use fix#xxxx（xxxx is issue number）

**Comment by tooo-bad** [comment]
Okay, I will resubmit the PR.

**Comment by tooo-bad** [comment]
I have added a newline, please check.

File: eventmesh
Comment (Pil0tXia) [inline eventmesh:1]: @tooo-bad It seems that you've added a git submodule. Please remove it.

**Review by Pil0tXia — CHANGES_REQUESTED** [review]

File: eventmesh
Comment (tooo-bad) [reply eventmesh:1]: I have searched the config file for no information about submodules, and there is no .gitmodules file in the root directory.

**Pull request closed without merging**